app/jobs/replication/results_recorder_job.rb

```diff
@@ -27,17 +25,12 @@ class ResultsRecorderJob < ApplicationJob
     # @param [Integer] version
     # @param [String] s3_part_key
     # @param [String] delivery_class Name of the worker class that performed delivery
-    def perform(druid, version, s3_part_key, delivery_class) # rubocop:disable Lint/UnusedMethodArgument used as job.arguments.fourth in before_perform
+    def perform(druid, version, s3_part_key, delivery_class) # rubocop:disable Lint/UnusedMethodArgument # delivery_class used as job.arguments.fourth in before_perform
       part = zip_part!(s3_part_key)
       part.ok!
 
-      # log to event service if this part upload is the last one for the endpoint
+      # log to event service if all ZipParts are replicated for this endpoint
       create_zmv_replicated_event(druid) if zmv.reload.all_parts_replicated?
-
-      # only publish result if all of the parts replicated for all zip_endpoints
-      return unless zmvs.reload.all?(&:all_parts_replicated?)
-
-      publish_result(message(druid, version).to_json)
```

Comment:
there is no consumer for these, and hasn't been for 4 years, so removing this code.